themes/src/main/resources/theme/base/admin/messages/admin-messages_pt_BR.properties

```diff
@@ -0,0 +1,814 @@
+#encoding: utf-8
+consoleTitle=Console de administração do Keycloak
```

Comment:
I used this new v2.0.0 header (#encoding: utf-8) so I enconded the file in UTF-8 and used "normal" characters. I think this way it's a lot faster to write, and easier to read and maintain.